packages/cardhost/app/components/editor-pane.js

```diff
@@ -1,7 +1,42 @@
 import Component from '@glimmer/component';
 import { inject as service } from '@ember/service';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
 
 export default class EditorPane extends Component {
-  @service cssModeToggle
+  @service cssModeToggle;
+  @tracked markup;
+  @tracked css = this.args.model.isolatedCss;
 
+  constructor() {
+    super(...arguments)
+    // getting markup must be done in the constuctor to guarantee that it
+    // resolves before the code editor is rendered.
+    let cardMarkup = document.querySelector('.card-renderer-isolated--content')
+    this.markup = cardMarkup ? cardMarkup.innerHTML.toString().trim() : "";
```

Comment:
This is where we grab the HTML. The indentation is a little strange when you get it from the DOM.

Reply:
What's the issue that this resolves?

Reply:
Oh is it so that the code editor actually has the markup? Were you seeing cases where the code editor showed nothing because the `markup` property was set late?

Reply:
I didn't see any issues, but this work has to happen somewhere, and I think it should be the responsibility of the editor pane and not CodeEditor.